galaxy_api/api/utils.py

```diff
@@ -0,0 +1,45 @@
+from collections import namedtuple
+import re
+
+
+CollectionFilename = namedtuple("CollectionFilename", ["namespace", "name", "version"])
+
+FILENAME_REGEXP = re.compile(
+    r"^(?P<namespace>\w+)-(?P<name>\w+)-" r"(?P<version>[0-9a-zA-Z.+-]+)\.tar\.gz$"
+)
+VERSION_REGEXP = re.compile(r"""
```

Comment:
Not sure if this will be part of the permission pr or not, but just a note that galaxy-importer still uses semantic_version package for these checks, shouldnt be a problem as long as they match.